src/main/java/Storage.java

```diff
@@ -0,0 +1,59 @@
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    String filepath;
+
+    Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * loads a list from the initiated filepath
+     * @return formatted and read list
+     */
+    ArrayList<ListItem> load() {
+        try {
+            Scanner reader = new Scanner(new FileReader(filepath));
+            ArrayList<ListItem> tempArray = new ArrayList<>();
+            while(reader.hasNext()) {
+                String commandIn = reader.nextLine();
+                String[] commandArray = commandIn.split("@");
+                tempArray.add(new ListItem(commandArray[2], commandArray[1], commandArray[3]));
```

Comment:
After you split the command, you can consider declaring each array item as a String for better understanding later, rather than commandArray[x]